.github/parm/use_case_groups.json

```
@@ -122,7 +122,7 @@
   {
     "category": "s2s",
     "index_list": "1-3",
-    "run": false
+    "run": true
```

Turn off use case for pull request so other feature branches don't run these cases.
```suggestion
    "run": false
```